forge-gui/res/cardsfolder/i/intrepid_adversary.txt

```diff
@@ -10,4 +10,5 @@ SVar:X:Count$TriggerRememberAmount
 S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddPower$ Z | AddToughness$ Z | Description$ Creatures you control get +1/+1 for each valor counter on CARDNAME.
 SVar:Z:Count$CardCounters.VALOR
 DeckHas:Ability$LifeGain|Counters
+AI:RemoveDeck:All
```

Comment:
imo the AI can use this good, even if it would pay them for the max, it is still okay if the AI already has a board of creatures

Reply:
I didn't tested it recently but I saw it happen on Android once

Reply:
I can reproduce it... huh...

Reply:
I can't figure out why the AI will dump mana into Intrepid Adversary but not Bloodthirsty Adversary or the Tranquil Frillback 😖